### PR TITLE
Update references in non-normative docs

### PR DESCRIPTION
**Pull request by fstrr**
- Replace reference to focus section of HTML 4.01 spec with reference to focus section of the HTML living standard in non-normative documents
- correct uses of lists to be either unordered or ordered as appropriate
- correct relative links

### CODE REVIEW

**Comment by mgifford** [comment]
@fstrr it seems like this is a patch that covers more than what is in the initial post. Could we get this updated to cover some of the new content that seems to be added? The description makes it sounds like a straight forward shift in existing content, but I do think there is more in it than this.

**Comment by mbgower** [comment]
> @fstrr it seems like this is a patch that covers more than what is in the initial post. Could we get this updated to cover some of the new content that seems to be added? The description makes it sounds like a straight forward shift in existing content, but I do think there is more in it than this.

@mgifford can you cite an example of what you're talking about?
@fstrr if you are aware and agree, can you please include those differences in the summary of the PR?

File: understanding/21/target-size-enhanced.html [hunk 1]
```diff
@@ -56,14 +56,14 @@ <h3>Benefits</h3>
   	<section id="examples">
   	  <h2>Examples</h2>
       <ul>      
-        <li><strong>Example 1: Buttons</strong><br/> Three buttons are on-screen and the touch target area of each button is 44 by 44 CSS pixels.</li>
-        <li><strong>Example 2: Equivalent target</strong><br/> Multiple targets are provided on the page that perform the same function. One of the targets is 44 by 44 CSS pixels. The other targets do not have a minimum touch target of 44 by 44 CSS pixels.</li>
-        <li><strong>Example 3: Anchor Link</strong><br/> The target is an in-page link and the target is less than 44 by 44 CSS pixels.</li>
-        <li><strong>Example 4: Text Link in a paragraph</strong><br/> Links within a paragraph of text have varying touch target dimensions. Links within
+        <li><strong>Example 1: Buttons</strong><br> Three buttons are on-screen and the touch target area of each button is 44 by 44 CSS pixels.</li>
```
Comment (mbgower) [inline understanding/21/target-size-enhanced.html:59]:
```suggestion
        <li><strong>Example 1: Buttons</strong><br/> Three buttons are on-screen and the touch target area of each button is 44 by 44 CSS pixels.</li>
```

File: understanding/21/target-size-enhanced.html [hunk 2]
```diff
@@ -56,14 +56,14 @@ <h3>Benefits</h3>
...
+        <li><strong>Example 2: Equivalent target</strong><br> Multiple targets are provided on the page that perform the same function. One of the targets is 44 by 44 CSS pixels. The other targets do not have a minimum touch target of 44 by 44 CSS pixels.</li>
```
Comment (mbgower) [inline understanding/21/target-size-enhanced.html:60]:
```suggestion
        <li><strong>Example 2: Equivalent target</strong><br/> Multiple targets are provided on the page that perform the same function. One of the targets is 44 by 44 CSS pixels. The other targets do not have a minimum touch target of 44 by 44 CSS pixels.</li>
```

File: understanding/21/target-size-enhanced.html [hunk 3]
```diff
@@ -56,14 +56,14 @@ <h3>Benefits</h3>
...
+        <li><strong>Example 3: Anchor Link</strong><br> The target is an in-page link and the target is less than 44 by 44 CSS pixels.</li>
```
Comment (mbgower) [inline understanding/21/target-size-enhanced.html:61]:
```suggestion
        <li><strong>Example 3: Anchor Link</strong><br/> The target is an in-page link and the target is less than 44 by 44 CSS pixels.</li>
```

File: understanding/21/target-size-enhanced.html [hunk 4]
```diff
@@ -56,14 +56,14 @@ <h3>Benefits</h3>
...
+        <li><strong>Example 4: Text Link in a paragraph</strong><br> Links within a paragraph of text have varying touch target dimensions. Links within
```
Comment (mbgower) [inline understanding/21/target-size-enhanced.html:62]:
```suggestion
        <li><strong>Example 4: Text Link in a paragraph</strong><br/> Links within a paragraph of text have varying touch target dimensions. Links within
```

File: understanding/21/target-size-enhanced.html [hunk 5]
```diff
@@ -56,14 +56,14 @@ <h3>Benefits</h3>
...
            paragraphs of text do no need to meet the 44 by 44 CSS pixels requirements.</li>
-        <li><strong>Example 5: Text Link in a sentence</strong><br/> A text link that is in a sentence is excluded and does not need to meet the 44 by 44 CSS pixel requirement. If the text link is the full sentence, then the text link target touch area does need to meet the 44 by 44 CSS pixels.</li>
-        <li><strong>Example 6: Footnote</strong><br/> A footnote link at the end of a sentence does not need to meet the 44 by 44 CSS pixels requirements. The footnote at the end of the sentence is considered to be part of the sentence.</li>
-        <li><strong>Example 7: Help icon</strong><br/> A help icon within or at the end of a sentence does not need to meet the 44 by 44 CSS pixels requirements. The icon at the end of the sentence is considered to be part of the sentence.</li>
+        <li><strong>Example 5: Text Link in a sentence</strong><br> A text link that is in a sentence is excluded and does not need to meet the 44 by 44 CSS pixel requirement. If the text link is the full sentence, then the text link target touch area does need to meet the 44 by 44 CSS pixels.</li>
```
Comment (mbgower) [inline understanding/21/target-size-enhanced.html:64]:
```suggestion
        <li><strong>Example 5: Text Link in a sentence</strong><br/> A text link that is in a sentence is excluded and does not need to meet the 44 by 44 CSS pixel requirement. If the text link is the full sentence, then the text link target touch area does need to meet the 44 by 44 CSS pixels.</li>
```

File: understanding/21/target-size-enhanced.html [hunk 6]
```diff
@@ -56,14 +56,14 @@ <h3>Benefits</h3>
...
+        <li><strong>Example 6: Footnote</strong><br> A footnote link at the end of a sentence does not need to meet the 44 by 44 CSS pixels requirements. The footnote at the end of the sentence is considered to be part of the sentence.</li>
```
Comment (mbgower) [inline understanding/21/target-size-enhanced.html:65]:
```suggestion
        <li><strong>Example 6: Footnote</strong><br/> A footnote link at the end of a sentence does not need to meet the 44 by 44 CSS pixels requirements. The footnote at the end of the sentence is considered to be part of the sentence.</li>
```

File: understanding/21/target-size-enhanced.html [hunk 7]
```diff
@@ -56,14 +56,14 @@ <h3>Benefits</h3>
...
+        <li><strong>Example 7: Help icon</strong><br> A help icon within or at the end of a sentence does not need to meet the 44 by 44 CSS pixels requirements. The icon at the end of the sentence is considered to be part of the sentence.</li>
```
Comment (mbgower) [inline understanding/21/target-size-enhanced.html:66]:
```suggestion
        <li><strong>Example 7: Help icon</strong><br/> A help icon within or at the end of a sentence does not need to meet the 44 by 44 CSS pixels requirements. The icon at the end of the sentence is considered to be part of the sentence.</li>
```

**Comment by mbgower** [comment]
@mgifford since this has good support from the AGWG review, and you have not provided any examples, I am merging. If you continue to think this PR has introduced an issue, please open a new issue to resolve.

**Pull request merged**